src/lib/marlin_plonk_bindings/js/node_js/dune

```diff
@@ -0,0 +1,26 @@
+(library
+ (name node_backend)
+ (js_of_ocaml
+  (flags (:include flags.sexp))
+  (javascript_files node_backend.js))
+ (instrumentation (backend bisect_ppx))
+ (preprocess (pps ppx_version js_of_ocaml-ppx)))
+
+(rule
+ (targets
+   plonk_wasm_bg.wasm.d.ts
+   plonk_wasm_bg.wasm
+   plonk_wasm.d.ts
+   plonk_wasm.js
+   flags.sexp)
+ (deps
+  (source_tree ../../wasm)
+  (source_tree ../../../marlin))
+ (action
+  (progn
+   (run chmod -R +w ../../wasm)
+   (run rustup run nightly-2021-02-11 wasm-pack build --target nodejs --out-dir ../js/node_js ../../wasm)
+   ;;(pipe-stdout ;; Inject fragment after the 3rd line to fix compilation
```

Comment:
Should this be deleted?

Reply:
This is needed for the other bindings (follow-up PR), I'd left it here because I kept clobbering it when git stashing.